### DISCUSSION TITLE
Announcing Sionna v0.10

### DISCUSSION THREAD

**Discussion by faycalaa** [NVlabs/sionna#34] [Announcements]
We are happy to announce the Sionna v0.10 release:

## New features:

- Universal FEC encoder for binary linear codes
- Utility functions for FEC to
  - Convert parity-check matrix to generator matrix
  - Make generator matrix systematic
  - Generate dense Polar parity-check and generator matrices
  - Verify that generator and parity check matrix are orthogonal in GF(2)
- Linear interpolator for OFDM channel estimation (with optional time-averaging)
- MIMO maximum likelihood detector that computes either LLRs on bits or logits on constellation points
- SymbolSource layer for sampling arbitrary constellations
- PAMSource layer for sampling PAM constellations
- The SymbolSource, PAMSource, QAMSource, and Mapper optionally return the indices of the constellation points and/or the bit labels
- Utility layer to compute logits on constellation points from LLRs on bits (with optional priors)
- Utility layer to compute the mean and variance of a constellation given logits on the points
- Addition of integration tests

 
## Fixes:

- (#25) XLA compatibility with TF2.9 for the BP decoder
- (#29) SNR property in PlotBER
- Adds support for TF2.9
- Minor bug fixes
- Typos in the documentation
